frontend/src/utils/error.ts

```diff
@@ -0,0 +1,12 @@
+import notify from "@/components/base/molecules/Notify";
+
+
+export function notifyError(baseMessage: string, err?: any, defaultMessage? : string) {
+  const message =
+  err?.message ||
+  err?.data?.error ||
```

Comment:
as per our last discussion, is this still required?

Reply:
Yes rishi told to keep like this so because we didn't actually find the reason why those 2 lines were used for handling but we are using function now so if any future we want to remove we can just remove from here only.